index.html

```diff
@@ -1,200 +1,259 @@
-                    <img src="images/sampleproject.svg" alt="">
-                    <div class="info">
-                        <h3> Profesional Art Printing Data </h3>
-                        <p> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
-                        <ul class="container tags"> 
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> html </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> bootstrap </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> Ruby </a></li>
-                        </ul>
-                    </div>
-                    <a rel="noopener noreferrer" href="#" class="interaction" aria-label="Project">See Project </a>
-                </div>
-                <div class="card other">
-                    <img src="images/sampleproject.svg" alt="">
-                    <div class="info">
-                        <h3> Profesional Art Printing Data </h3>
-                        <p> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
-                        <ul class="container tags"> 
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> html </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> bootstrap </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> Ruby </a></li>
-                        </ul>
-                    </div>
-                    <a rel="noopener noreferrer" href="#" class="interaction" aria-label="Project">See Project </a>
-                </div>
-                <div class="card other">
-                    <img src="images/sampleproject.svg" alt="">
-                    <div class="info">
-                        <h3> Profesional Art Printing Data </h3>
-                        <p> A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
-                        <ul class="container tags"> 
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> html </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> bootstrap </a></li>
-                            <li class="textPlaceholder tag"><a rel="noopener noreferrer" href="#" aria-label=""> Ruby </a></li>
-                        </ul>
-                    </div>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale= 1.0">
+    <title> My portfolio </title>
```

Comment:
- [ ] [OPTIONAL]: To keep your project even more professional, you should consider adding a more descriptive title, what about having "your name's portfolio", for instance?